### PR TITLE
rakeshnrao adds hometown of San_Ramon

### PR DESCRIPTION
**Pull request by rakeshnrao**
Hi all, 👋 

Add some text that you want to invite others for comments

San Ramon is awesome. You should visit at least once.

Rax

### CODE REVIEW

**Comment by thirugnanamsubbiah** [comment]
@rakeshnrao Sure I would stop by

**Comment by joconner** [comment]
Will give you a call if ever in San Ramon!

File: rakeshnrao-san_ramon.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# San Ramon, CA
+
+### Land of earth and wind
+
+### Pleanty of open space
+
```
Comment (DanSewell) [inline rakeshnrao-san_ramon.md]: Trying to add a line comment?  Code Comment?  Not sure what this is called.  but I am trying to add something in here

File: rakeshnrao-san_ramon.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# San Ramon, CA
+
```
Comment (DanSewell) [inline rakeshnrao-san_ramon.md]: Looks like it stays where I clicked

**Comment by thirugnanamsubbiah** [comment]
👍  Good to Merge

**Comment by joconner** [comment]
👍

**Pull request merged**